### PR TITLE
Add installation instructions for NixOS

### PR DESCRIPTION
**Pull request by bernardpaulus**
Hello,

I thought about adding installation instructions for [NixOS](https://nixos.org/).

I checked the code of conduct and the readme, but if I missed any other guideline, don't hesitate to notify me  and I'll do the necessary :)

### CODE REVIEW

File: getting-started/installation.md [hunk 1]
```diff
@@ -166,6 +166,14 @@ Built-in, no need to install.
 sudo emerge app-containers/podman
 ```
 
+#### [NixOS](https://nixos.org/)
+
+```nix
+virtualisation.podman.enable = true;
```
Comment (lsm5) [inline getting-started/installation.md:172]: I have no idea about nixos, but just double checking that there are no other preceeding / succeeding commands to install podman.

**Review by lsm5 — COMMENTED** [review]
LGTM, assuming that's all we need to install podman on nixos. This seems to be consistent with the nixos wiki link, but that's the extent of my understanding.

I'd also like to double check if the podman nixos package is actively being maintained. We had some issues with the skopeo nixos package in the past https://github.com/containers/skopeo/pull/1463

File: _config.yml [hunk 2]
```diff
@@ -59,6 +59,11 @@ authors:
     web: https://balagetech.com
     twitter:
     github: abalage
+  bernardpaulus:
+    name: Bernard Paulus
+    display_name: Bernard Paulus
+    email: bernard.paulus@gmail.com
+    github: bernardpaulus
```
Comment (TomSweeneyRedHat) [inline _config.yml:66]: FWIW, this is only needed if you plan to write a blog post for the site, and we've archived the posts for everything but release announcements.  Doesn't hurt, and we'll keep it, just thought I'd whip up a quick comment.  blogs.podman.io is the new home for the blog posts pertaining to podman fwiw.

File: getting-started/installation.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply getting-started/installation.md:172]: Ditto, and this looks reasonable, and the like is especially good.

**Comment by TomSweeneyRedHat** [comment]
LGTM, thansk for the contribution @bernardpaulus !

**Pull request merged**